pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -223,4 +224,7 @@ func SetDefaults_KubeletConfiguration(obj *kubeletconfigv1beta1.KubeletConfigura
 	if obj.EnforceNodeAllocatable == nil {
 		obj.EnforceNodeAllocatable = DefaultNodeAllocatableEnforcement
 	}
+	if obj.VolumePluginDir == "" {
```

Comment:
Was there any pre-existing validation code (maybe spread around the codebase) that we'd need to add to `pkg/kubelet/apis/config/validation.go`?

Reply:
I did a basic search for validations for VolumePluginDir config, I couldn't find any.